packages/integration-functions/document-metadata-fo-integration/index.js

```diff
@@ -3,43 +3,12 @@ const VALID_SCAN_STATUSES = ['scanned'];
 const VALID_PUBLISHED_STATUSES = ['published', 'archived'];
 
 /**
- * @typedef {Object} ServiceBusMessage
- * @property {DocumentPayload} body
+ * @typedef {import('@pins/common/schema/documentMetadata.js').DocumentMetadata} DocumentMetadata
```

Comment:
Does fo stand front office in this file name?

Reply:
Talking about "packages/integration-functions/document-metadata-fo-integration/index.js", not the target of this import btw, didn't know where else to attach this

Reply:
yes, was just following the naming from appeals-case-data, not sure it's really needed as it lives in a function app with that name already: pins-func-appeals-fo-integration

didn't do it with listed building ones as there is no equivalent in Back Office